.travis.yml

```diff
@@ -42,8 +42,9 @@ install:
   # Customise the testing environment
   # ---------------------------------
   - conda config --add channels scitools
-  - PACKAGES="$PACKAGES cython pillow nose pep8 proj4 pyshp shapely six requests pyepsg owslib"
+  - PACKAGES="$PACKAGES cython pillow nose pytest pep8 proj4 pyshp shapely six requests pyepsg owslib"
```

Comment:
No, it's still need for matplotlib's [image comparison decorator](https://travis-ci.org/SciTools/cartopy/jobs/121297187#L752-L757).